spec/poros/movie_spec.rb

```diff
@@ -1,56 +1,97 @@
 require "rails_helper"
 
 RSpec.describe Movie do
-  #set up VCR and change before block 
```

Comment:
Probably a conflict here as I kept the lines that were removed. We can look at in refactoring.

Edit: discussed in pair and merge conflict resolved.